packages/sites/ortho-site/webapp/wdkCustomization/js/client/records/GroupStats.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { useEffect, useRef, useState } from 'react';
+import { RecordTable } from './SequenceRecordClasses.SequenceRecordClass';
+import { RecordTableProps, WrappedComponentProps } from './Types';
+
+import eval_hist_img from '../../../images/eval-hist.png';
```

Comment:
I tried to set up a path alias thingy in tsconfig.json (there was one already in webpack.config.js - but we don't seem to be using it) but I couldn't get it to work :-(

Reply:
OK, I'll do that! Thanks!